components/admin_console/system_user_detail/team_list/abstract_list.jsx

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why was this file cloned? Any specific reason? Maybe better to extend the original?